punchcut/src/error.rs

```diff
@@ -0,0 +1,52 @@
+use read_fonts::{types::GlyphId, ReadError};
```

Comment:
We should have a common glyphname too! https://github.com/googlefonts/fontations/issues/201

Reply:
This would likely need to be feature gated as `font-types` can't depend on external libraries for `smol_str` and also doesn't depend on `std`, so `Arc` is out. I was thinking a `fontc-types` crate might be useful for the compiler.